index.js

```diff
@@ -1790,6 +1790,29 @@
   S.unlines =
   def('unlines', [List], R.compose(R.join(''), R.map(R.concat(_, '\n'))));
 
+  //# unfoldr :: (b -> Maybe (a, b)) -> b -> [a]
+  //.
+  //. The unfoldr function is a `dual' to reduce: while reduce "reduces" a list
+  //. to a summary value, unfoldr builds a list from a seed value. The function
+  //. takes the element and returns an instance of Nothing if it is done
+  //. producing the list or returns an instance of Just [a, b], in which case,
+  //. a is a prepended to the list and b is used as the next element in a
+  //. recursive call.
+  //.
+  //. ```javascript
+  //. > S.unfoldr(function(n){return n >= 5 ? S.Nothing() : S.Just([n, n + 1]);}, 1)
```

Comment:
When I write recursive functions I usually write the base case first, then the recursive case next.